include/iotc_error.h

```diff
@@ -21,118 +21,105 @@
+  /** @cond Numeric code: 47 */ IOTC_ELEMENT_NOT_FOUND, /** @endcond */
+  /** @cond Numeric code: 48 */ IOTC_SERIALIZATION_ERROR, /** @endcond */
+  /** @cond Numeric code: 49 */ IOTC_TRUNCATION_WARNING, /** @endcond */
+  /** @cond Numeric code: 50 */ IOTC_BUFFER_OVERFLOW, /** @endcond */
+  /** @cond Numeric code: 51 */ IOTC_THREAD_ERROR, /** @endcond */
+  /** @cond Numeric code: 52 */ IOTC_NULL_CONTEXT, /** @endcond */
+  /** @cond Numeric code: 53 */ IOTC_NULL_WILL_TOPIC, /** @endcond */
+  /** @cond Numeric code: 54 */ IOTC_NULL_WILL_MESSAGE, /** @endcond */
+  /** @cond Numeric code: 55 */ IOTC_NO_MORE_RESOURCE_AVAILABLE, /** @endcond */
+  /** @cond Numeric code: 56 */ IOTC_FS_RESOURCE_NOT_AVAILABLE, /** @endcond */
+  /** @cond Numeric code: 57 */ IOTC_FS_ERROR, /** @endcond */
+  /** The memory limiter module isn't installed. To enable the memory limiter, set the <a href="../../../porting_guide.md#development-flags"><code>memory_limiter</code></a> flag in a <a href="../../../porting_guide.md#config">CONFIG</a> argument. <br> Numeric code: 58 */ IOTC_NOT_SUPPORTED,
+  /** The event processor stopped because the client application {@link iotc_events_stop() shut down the event engine}. Numeric code: 59 */ IOTC_EVENT_PROCESS_STOPPED,
+  /** @cond Numeric code: 60 */ IOTC_STATE_RESEND, /** @endcond */
+  /** @cond Numeric code: 61 */ IOTC_NULL_HOST, /** @endcond */
+  /** @cond Numeric code: 62 */ IOTC_TLS_FAILED_CERT_ERROR, /** @endcond */
+  /** @cond Numeric code: 63 */ IOTC_FS_OPEN_ERROR, /** @endcond */
+  /** @cond Numeric code: 64 */ IOTC_FS_OPEN_READ_ONLY, /** @endcond */
+  /** @cond Numeric code: 65 */ IOTC_FS_READ_ERROR, /** @endcond */
+  /** @cond Numeric code: 66 */ IOTC_FS_WRITE_ERROR, /** @endcond */
+  /** @cond Numeric code: 67 */ IOTC_FS_CLOSE_ERROR, /** @endcond */
+  /** @cond Numeric code: 68 */ IOTC_FS_REMOVE_ERROR, /** @endcond */
+  /** @cond Numeric code: 69 */ IOTC_NULL_PROJECT_ID_ERROR, /** @endcond */
+  /** The private key isn't signed with ES256. Numeric code: 70 */ IOTC_ALG_NOT_SUPPORTED_ERROR,
+  /** @cond Numeric code: 71 */ IOTC_JWT_FORMATTION_ERROR, /** @endcond */
+  /** @cond Numeric code: 72 */ IOTC_JWT_PROJECTID_TOO_LONG_ERROR, /** @endcond */
+  /** @cond Numeric code: 73 */ IOTC_NULL_DEVICE_PATH_ERROR, /** @endcond */
+  /** The buffer is too small for the data. Numeric code: 74 */ IOTC_BUFFER_TOO_SMALL_ERROR,
+  /** The buffer for storing formatted and signed JWTs is null. Numeric code: 75 */ IOTC_NULL_KEY_DATA_ERROR,
+  /** @cond Numeric code: 76 */ IOTC_NULL_CLIENT_ID_ERROR, /** @endcond */
 
-  IOTC_ERROR_COUNT /* add above this line, and this should always be last. */
+  IOTC_ERROR_COUNT /* Add errors above this line; this should always be last line. */
 } iotc_state_t;
 
 /**
- * @def
- * @brief undefine to enable the translation of iotc_state_t to human readable
- * strings
+ * @brief A flag to translate between {::iotc_state_t state messages} to their numeric codes.
```

Comment:
This appears weird on the .html page (, probably "@link" is missing from the beginning). The curly braces are there, but the link works to iotc_state_t.

Reply:
Done